### ISSUE TITLE
Cannot choose VLC as video player in SE v3.5.14

### ISSUE THREAD

**Issue by MariuxReloaded** [SubtitleEdit/subtitleedit#4135]
Hello everyone 👋
As the title of this thread says and as you can see in my screenshot, I've just formatted my laptop and installed the new Windows 10 May 2020 Update and, after reinstalling Subtitle Edit, I've found out that I can't choose which video player I want to use anymore 😥
Is there a fix for this?
Thank you very much in advance 👍

![Senza titolo-1](https://user-images.githubusercontent.com/46523668/79814897-ab861d80-837f-11ea-8f02-68509bde00f6.jpg)

**Comment by niksedk** [comment]
@MariuxReloaded: You're running a 64-bit OS, so you need the 64-bit VLC: https://get.videolan.org/vlc/3.0.8/win64/vlc-3.0.8-win64.exe

But I would rather recommend that you click on "Download mpv lib" - will give you a precise video/audio position + better preview on the video player.

**Comment by Signy13** [comment]
@niksedk : "Download mpv lib" is not working on Linux (Wine 5.0 + .NET) unfortunately
![SubtitleEdit_70_mpv_lib](https://user-images.githubusercontent.com/18901924/79831508-dab08500-83a7-11ea-9a47-45dbf71bd7cb.png)
Could you please add info (where to download the library and where to save it) into the download error pop-up? And it would be nice if there was a possibility to copy the link to the library into clipboard...

**Comment by MariuxReloaded** [comment]
> @MariuxReloaded: You're running a 64-bit OS, so you need the 64-bit VLC: https://get.videolan.org/vlc/3.0.8/win64/vlc-3.0.8-win64.exe
> 
> But I would rather recommend that you click on "Download mpv lib" - will give you a precise video/audio position + better preview on the video player.

OMG! You're absolutely right! 🤦‍♂️
I've downloaded and installed the 64 bit version of VLC for years, after formatting my laptop, but this time, maybe due to tiredness or because I I did it in too much of a hurry, I completely forgot to click on the arrow near "Download VLC" and choose the installer for the 64 bit version.
Now everything is ok, of course.
I'm deeply sorry for this inconvenience and thank you very much 👍
Dear VLC, why don't you change the layout of your Downloads page, instead of continuing to hide the other choices under the 32 bit Windows version? 😒
